tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -8,6 +8,8 @@
 from datadog_checks.base.errors import CheckException
 from datadog_checks.base.utils.time import get_precise_time
 
+DEFAULT_IP_CACHE_DURATION = 120
```

Comment:
```suggestion
```
Default needs to be current behaviour to avoid unexpected changes to existing users